assets/js/googlesitekit/components-gm3/Checkbox.js

```diff
@@ -89,20 +89,7 @@ export default function Checkbox( {
 		};
 	}, [ checked, onChange, onKeyDown ] );
 
-	// TODO: Remove `onLabelClick` once the `md-checkbox` component is updated to be a form-associated custom element.
-	const onLabelClick = () => {
-		const { current } = ref;
-
-		if ( ! current || disabled ) {
-			return;
-		}
-
-		current.checked = ! current.checked;
-
-		current.dispatchEvent(
-			new Event( 'change', { bubbles: true, cancelable: true } )
-		);
-	};
+	const labelID = `${ id }-label`;
```

Comment:
Noting that the `element-internals-polyfill` package (introduced in this PR here to to polyfill `attachInternals()` in JSDom) [sets `aria-labelledby`](https://github.com/calebdwilliams/element-internals-polyfill/blob/8aa2b3d51db00d32085153b003fdffcd527eebcd/src/utils.ts#L75-L79) reminded me that we should do that ourselves for non-polyfilled production scenarios.